src/app/services/router-guards.service.ts

```diff
@@ -40,18 +40,7 @@ export class RouterGuards implements CanActivate {
     // restrict access to audiologist pages
     if (url === '/audiologist') {
       let pin = Utilities.getSessionStorage('audiologist-pin');
```

Comment:
You replaced setting 'audiologist-pin', with setting 'userId' and 'sessionId' in audiologist-login component. So pin will always be null.

Reply:
The plan is to test this router guard with a sessionId check.

Reply:
I meant to remove that call.